src/BSeq.jl

```diff
@@ -0,0 +1,209 @@
+"""
+Solve Bethe-Slapter equation
+"""
+
+module BSeq
+
+using ..Parameter, ..Convention, ..LegendreInteraction
+using ..Parameters, ..GreenFunc, ..Lehmann, ..CompositeGrids
+
+const freq_sep = 0.01
+
+function initFR(Euv, rtol, sgrid, param)
+    @unpack β, me, μ = param
+    Ω_c = freq_sep
+
+    wn_mesh = GreenFunc.ImFreq(β, FERMION; Euv=Euv, rtol=rtol, symmetry=:pha)
+    R_freq = GreenFunc.MeshArray(wn_mesh, sgrid; dtype=Float64)
+    F_freq = similar(R_freq)
+    for ind in eachindex(R_freq)
+        ni, ki = ind[1], ind[2]
+        ωn, k = matfreq(wn_mesh)[ni], sgrid[ki]
```

Comment:
matfreq(wn_mesh)[ni] --> wn_mesh[ni] should directly return real-valued Matsubara frequency

Reply:
Agreed.